nbconvert/exporters/pdf.py

```diff
@@ -163,9 +163,9 @@ def from_notebook_node(self, nb, resources=None, **kw):
         )
         # set texinputs directory, so that local files will be found
         if resources and resources.get('metadata', {}).get('path'):
-            self.texinputs = resources['metadata']['path']
+            self.texinputs = str_to_unicode(resources['metadata']['path'])
```

Comment:
The general rule with unicode is that we should convert this as soon as possible, i.e. where it's passed in to nbconvert, rather than here. Have a look in `Exporter.from_filename`.